src/helpers/utilities.ts

```diff
@@ -451,6 +451,49 @@ export const omitFlatten = <T extends object, K extends keyof T>(
   return newObj;
 };
 
+/**
+
+ * @desc Converts the first character of string to upper case and the remaining to lower case
+ */
+export const capitalize = (string?: string | undefined) => {
+  if (!string) return '';
+  return string
+    ? string.charAt(0).toUpperCase() + string.slice(1).toLowerCase()
+    : '';
+};
+
+/**
+ * @desc Converts the first character of string to upper case and the remaining to lower case
+ */
+export const startCase = (string?: string | undefined): string => {
+  if (!string) return '';
+  return string
+    ? string.charAt(0).toUpperCase() + string.slice(1).toLowerCase()
+    : '';
+};
```

Comment:
@perunt aren't these the same thing?

Reply:
Start case should start every word with a big letter. Maybe we don't need this?

Reply:
I use old changes from [this](https://github.com/rainbow-me/rainbow/pull/3602) PR
I will update to [that](https://github.com/rainbow-me/rainbow/pull/3602/files#diff-dfbbddbff81e448260b519c9739b639b428c4ba3bb3a4aab4e6e82eab651c367) one